stratum/hal/bin/ipdk/ipdk_main.cc

```diff
@@ -0,0 +1,119 @@
+// Copyright 2018-2019 Barefoot Networks, Inc.
```

Comment:
no bazel BUILD file in this directory

Reply:
We're not ready to build IPDK with Bazel. Omitting the BUILD file causes Bazel to ignore the directory.